src/extension.ts

```diff
@@ -1,65 +1,43 @@
-import * as vscode from 'vscode'
+import * as vscode from 'vscode';
 import { decorate, userDecorate } from './decorator';
-import { userDefinedHeaderProperties } from "./userDefinedHeaderProperties";
-
-export function activate(context: vscode.ExtensionContext) {
-    console.log('Congratulations, your extension "vscode-ramarkdown-header-coloring" is now active!');
-
-    // let userDefinedHeaderColor;
-    // let userDefinedHeaderColor: Boolean = vscode.workspace.getConfiguration('markdown-header-coloring').get<Boolean>('userDefinedHeaderColor');
-    let userDefinedHeaderColor: userDefinedHeaderProperties = vscode.workspace.getConfiguration('markdown-header-coloring').get<object>('userDefinedHeaderColor');
-
-    
-    // if (!vscode.window.activeTextEditor) {
-        
-    // } else {
-    //     // editor = vscode.window.activeTextEditor;
-    //     if (vscode.window.activeTextEditor.document.languageId == 'markdown') {
-    //         (userDefinedHeaderColor.enabled === false) ? decorate() : userDecorate();
-    //         // userDecorate();
-    //     }    
-    // }
+import { userDefinedHeaderProperties } from './userDefinedHeaderProperties';
 
+// Helper to check for markdown or quarto
+function isMarkdownOrQuarto(editor: vscode.TextEditor): boolean {
+    return ['markdown', 'quarto'].includes(editor.document.languageId);
+}
```

Comment:
Reckon this could be configurable, as a list of enabled languages?
```jsonc
// settings.json
"markdown-header-coloring.enabledLanguages": [
    "markdown",
    "quarto",
    "rmd"
]
```

The language checking function could become something like
```suggestion
function isLanguageEnabled(editor: vscode.TextEditor): boolean {
    const enabledLanguages = vscode.workspace.getConfiguration('get<string[]>('markdown-header-coloring').get<string[]>('enabledLanguages')
    return enabledLanguages.includes(editor.document.languageId);
}
```